File: digdag-cli/src/main/resources/digdag/cli/init_examples/python/scripts/myclass.py
from dateutil import parser
from random import randint

class MyClass(object):
    def __init__(self):
        pass

    def print_time(self, label, time):
        print([label, parser.parse(time)])

    def say_something(self, order, animal):
        print("""%s %s said "My favorite number is %d" """ % (order, animal, randint(0, 9)))

File: digdag-cli/src/main/resources/digdag/cli/init_examples/ruby/scripts/myclass.rb
require 'time'

class MyClass
  def print_time(label:, time:)
    p [label, Time.parse(time)]
  end

  def say_something(order:, animal:)
    puts %Q|#{order} #{animal} said "My favorite number is #{Random.rand(10)}"|
  end
end
